Source/Core/DolphinQt2/HotkeyScheduler.cpp

```diff
@@ -159,17 +159,17 @@ void HotkeyScheduler::Run()
       if (IsHotkey(HK_EXIT))
         emit ExitHotkey();
 
+      auto& settings = Settings::Instance();
+
       // Volume
       if (IsHotkey(HK_VOLUME_DOWN))
-        AudioCommon::DecreaseVolume(3);
+        settings.SetVolume(settings.GetVolume() - 3);
```

Comment:
This is not the same as calling `DecreaseVolume`: that function unmutes, clamps the volume to a normal range (0 to 100), and calls `UpdateSoundStream` to have the volume change take effect.

Maybe you could do something like this:

```c++
void Settings::DecreaseVolume(int decrease) {
  AudioCommon::DecreaseVolume(decrease);
  emit VolumeChanged();
}
```

?